client/build.rs

```diff
@@ -30,6 +30,17 @@ fn generate_grpc_geyser() -> anyhow::Result<()> {
                 .server_streaming()
                 .build(),
         )
+        .method(
+            Method::builder()
+                .name("subscribe_jup")
+                .route_name("SubscribeJup")
```

Comment:
```suggestion
        .method(
            Method::builder()
                .name("patchable_subscribe")
                .route_name("PatchableSubscribe")
```

Or just subscribe_v2 if want the add remove semantics to be the new standard.

What is a name we can use that isn't jup specific for releasing this to the greater community?